db/cedar_db_utils.go

```diff
@@ -0,0 +1,52 @@
+package db
+
+import (
+	"context"
+	"errors"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/mongodb/anser/db"
+)
+
+type cedarShimFactoryImpl struct {
+	env evergreen.Environment
+	db  string
+}
+
+func GetCedarGlobalSessionFactory() SessionFactory {
```

Comment:
nit: could we add a comment here + to `GetGlobalSessionFactory()` to mention that the other global session factory is for the Evergreen DB, while this is for the Cedar DB?